libs/core/src/lib/dynamic-page/dynamic-page-header/header/dynamic-page-header.component.spec.ts

```diff
@@ -35,6 +35,38 @@ class TestComponent {
     constructor(public dynamicPageService: DynamicPageService) {}
 }
 
+@Component({
+    template: `
+        <fd-dynamic-page-header [title]="title" [subtitleTemplate]="subtitleTemplate">
+            <fd-breadcrumb>
+                <fd-breadcrumb-item>
+                    <a fd-link [attr.href]="'#'">Men</a>
+                </fd-breadcrumb-item>
+                <fd-breadcrumb-item>
+                    <a fd-link [attr.href]="'#'">Shoes</a>
+                </fd-breadcrumb-item>
+            </fd-breadcrumb>
+
+            <fd-dynamic-page-global-actions></fd-dynamic-page-global-actions>
+            <fd-dynamic-page-title-content></fd-dynamic-page-title-content>
+        </fd-dynamic-page-header>
+
+        <ng-template #subtitleTemplate>
```

Comment:
I would introduce some sort of structural directive inside content projection of `fd-dynamic-page-header` so developers wouldn't need to pass a reference as an input property of the component.

Reply:
+1 for this @fkolar

Reply:
@N1XUS, This was my earlier discussion.  what is the preference of passing custom template and the response was F-NGX uses @Input and that F-NGX is consistent in this => @mikerodonnell89 

If its directive/component/ng-template  - this was my point. 

My preference would be simple ng-template.